src/test/java/unicash/testutil/TransactionUtil.java

```diff
@@ -34,7 +34,7 @@ public static String getTransactionDetails(Transaction transaction) {
         StringBuilder sb = new StringBuilder();
         sb.append(PREFIX_NAME).append(transaction.getName().fullName).append(" ");
         sb.append(PREFIX_TYPE).append(transaction.getType().type.getOriginalString()).append(" ");
-        sb.append(PREFIX_AMOUNT).append(transaction.getAmount().toString()).append(" ");
+        sb.append(PREFIX_AMOUNT).append(transaction.getAmount().originalString()).append(" ");
```

Comment:
Just curious but is this `originalString` going to be cached when we store to storage? If it is, no issues, but if it's not, this might be an issue in terms of "preserving" the value

Reply:
I'm not entirely sure what you mean, but this change was also discussed [here](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/96#discussion_r1367844482). If you are referring to the way Transactions are stored, in `JsonAdaptedTransaction` I noticed that an Amount's `amount` field is retrieved directly, maybe you could further clarify what you mean?

Reply:
Gotcha, for some reason `originalString` gives off the impression that the function returns either `$xx.xx` or `xx.xx` since original could refer to the original format that the amount came in.

If that's the case, maybe we can focus on making the method name more obvious with the purpose like `amountString()` or something similar so the code reads more smoothly?